Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1787,7 +1787,7 @@ namespace Tensile
             }
         }
 
-        return remainder >= get<1>(predictionCurve.back()) ? 0 : 1;
+        return remainder >= std::get<1>(predictionCurve.back()) ? 0 : 1;
```

Comment:
Did we confirm that the compiler warning goes away in rocblas?

Reply:
Confirmed.